src/Microsoft.DotNet.Build.Tasks/PackageFiles/versioning.props

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <Company Condition="'$(Company)' == ''">Microsoft Corporation</Company>
+    <Copyright Condition="'$(Copyright)' == ''">%A9 Microsoft Corporation.  All rights reserved.</Copyright>
+    <Description Condition="'$(Description)' == ''">$(AssemblyName)</Description>
+    <FileVersion Condition="'$(FileVersion)' == '' and '$(AssemblyFileVersion)' != ''">$(AssemblyFileVersion)</FileVersion>
+    <InformationalVersion Condition="'$(InformationalVersion)' == ''">$(AssemblyFileVersion)$(BuiltByString). Commit Hash%3A $(LatestCommit)</InformationalVersion>
+    <Product Condition="'$(Product)' == ''">Microsoft%AE .NET Framework</Product>
+    <Title Condition="'$(Title)' == ''">$(AssemblyName)</Title>
```

Comment:
The SDK calls this property `AssemblyTitle`:  https://github.com/dotnet/sdk/blob/8204b6b1f3ff2f9717c5d66ab903795271de2e9c/src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.GenerateAssemblyInfo.targets#L77-L78